### PR TITLE
Translation into Ukrainian

### PR DESCRIPTION
**Pull request by alexukua**
Revised, stylistically edited, translated into Ukrainian and structured according English version

### CODE REVIEW

File: locale/ru_RU/submission.xml [hunk 1]
```diff
@@ -18,9 +18,9 @@
...
 	<message key="submission.title">Заголовок статьи</message>
-	<message key="submission.title.tip"><![CDATA[Тип материала обычно что-то из «изображение», «текст» или другие типы мультимедиа, включая «программное обеспечение» или «интерактив». Пожалуйста, выберите один наиболее подходящий тип для вашего материала. Примеры можно найти на <a target="_blank" href="http://dublincore.org/documents/2001/04/12/usageguide/generic.shtml#type">http://dublincore.org/documents/2001/04/12/usageguide/generic.shtml#type</a>]]></message>
-	<message key="submission.submit.newSubmissionMultiple">Начать отправку нового материала в</message>
-	<message key="submission.submit.newSubmissionSingle">Новый материал</message>
+	<message key="submission.title.tip"><![CDATA[Тип материала обычно что-то из «изображение», «текст» или другие типы мультимедиа, такі як «програмне забезпечення» або «інтерактив». Будь ласка, виберіть один найбільш підходящий тип для вашого матеріалу. Приклади див. за посиланням <a target="_blank" href="http://dublincore.org/documents/2001/04/12/usageguide/generic.shtml#type">http://dublincore.org/documents/2001/04/12/usageguide/generic.shtml#type</a>]]></message>
```
Comment (ppv1979) [inline locale/ru_RU/submission.xml]: It's Ukrainian text added to Russian locale. Need to be fixed.

File: locale/ru_RU/submission.xml [hunk 2]
```diff
@@ -18,9 +18,9 @@
...
+	<message key="submission.submit.newSubmissionMultiple">Розпочати подання нового рукопису в</message>
```
Comment (ppv1979) [inline locale/ru_RU/submission.xml]: It's Ukrainian text added to Russian locale. Need to be fixed.

File: locale/ru_RU/submission.xml [hunk 3]
```diff
@@ -18,9 +18,9 @@
...
+	<message key="submission.submit.newSubmissionSingle">Новий рукопис</message>
```
Comment (ppv1979) [inline locale/ru_RU/submission.xml]: It's Ukrainian text added to Russian locale. Need to be fixed.

**Comment by mtub** [comment]
Thank you very much for contributing. I apologize for being so late with my review. From the first look of it, this looks really good. I have a few changes to suggest:

a) could you remove the non-uk_UA files from your PR? (.gitignore, fatal, git, and the Russian locale file)
b) we would like to maintain lists of contributors/translators on the wiki, see e.g. https://pkp.sfu.ca/wiki/index.php?title=OJS:_Russian_(ru_RU) or https://pkp.sfu.ca/wiki/index.php?title=OJS:_German_(de_DE). Could we move the name of the translator to the wiki and put a link to the wiki into the locale files instead? See e.g. https://github.com/pkp/ojs/blob/master/locale/ru_RU/admin.xml

**Comment by mtub** [comment]
Hi @alexukua , do you have an update for me on how you want to proceed?

**Comment by alexukua** [comment]
Hi @mtub, i resolved all conflict, you can accept pull request.

**Comment by mtub** [comment]
Hi @alexukua, I'd still request that you removed the .gitignore and the Russian files from the PR. Please tell me if I should grab the Ukrainian files and commit them myself. Also, I'd like to move the names of people involved in the translation to the wiki, as described above. Are you okay with this?

Finally, we have to look at https://github.com/pkp/ojs/pull/1341 (tagging @Vitaliy-1).  It would be good if you could work together or at least agree on translations.

**Comment by mtub** [comment]
I will now go on and pick only the uk_UA files from this PR and commit them. Will then compare with Vitaly's PR.

**Pull request closed without merging**